common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -161,6 +161,9 @@ private boolean handleUrl(final WebView view, final String url) {
         } else if (isInstallRequestUrl(formattedURL)) {
             Logger.info(TAG, "It is an install request");
             return processInstallRequest(view, url);
+        } else if (isWebCpUrl(formattedURL)) {
+            Logger.info(TAG, "It is a request from WebCP");
+            return processWebCpRequest(view, url);
```

Comment:
should we write this as a getUriHanlder method that returns a BiFunction that we call to handle the URI and view?